packages/sdk-client/tests/plugins/timezone/timezone.response.test.ts

```diff
@@ -0,0 +1,45 @@
+import { TimezoneResponse } from '../../../src';
+import { ResponsePluginContext } from '../../../src/plugins/core/response-plugin';
+import { Headers } from 'node-fetch';
+
+describe('Timezone response plugin', () => {
+
+  let context: ResponsePluginContext;
+  const TIMESTAMP_WITH_MISSING_TIMEZONE = '2024-01-09T15:50:24.000';
+  const TIMESTAMP_WITH_TIMEZONE = '2024-01-09T15:50:24.000Z';
+
+  beforeEach(() => {
+    context = {
+      operationId: 'GetCallResult',
+      apiName: '',
+      url: '',
+      requestOptions: {
+        headers: new Headers(),
+        basePath: '',
+      },
+    };
+  });
+
+  it('should update the timestamp if the timezone is missing', async () => {
+    const apiResponse = {
+      timestamp: TIMESTAMP_WITH_MISSING_TIMEZONE,
+    };
+    const plugin = new TimezoneResponse();
+    const runner = plugin.load(context);
+    const result = await runner.transform(apiResponse);
+
+    expect(result.timestamp).toBe(TIMESTAMP_WITH_TIMEZONE);
+  });
+
+  it('should NOT update the timestamp if the operationId if not listed', async () => {
+    context.operationId = 'notListedAsBuggy';
+    const apiResponse = {
+      timestamp: TIMESTAMP_WITH_MISSING_TIMEZONE,
+    };
+    const plugin = new TimezoneResponse();
+    const runner = plugin.load(context);
+    const result = await runner.transform(apiResponse);
+
+    expect(result.timestamp).toBe(TIMESTAMP_WITH_MISSING_TIMEZONE);
+  });
+});
```

Comment:
What happens when API is updated but SDK is not, 
Can you check for case when correct timezone is returned in response?

Reply:
New test added

Reply:
Not required but another test with a `+XX` timezone format could ensure regexp coverage